src/Symfony/Component/Translation/Bridge/Loco/composer.json

```diff
@@ -20,6 +20,9 @@
         "symfony/http-client": "^5.3",
         "symfony/translation": "^5.3"
     },
+    "require-dev": {
+        "symfony/config": "^4.4|^5.0"
```

Comment:
```suggestion
        "symfony/config": "^4.4|^5.2"
```
?

Reply:
I C&P'ed the constraint from `symfony/translation`. As @stof already mentioned, `symfony/translation` should manage what exact versions are compatible.